sql/core/src/main/scala/org/apache/spark/sql/jdbc/jdbc.scala

```diff
@@ -195,7 +195,9 @@ package object jdbc {
 
     override def getMinorVersion: Int = wrapped.getMinorVersion
 
-    override def getParentLogger: java.util.logging.Logger = wrapped.getParentLogger
+    def getParentLogger: java.util.logging.Logger =
```

Comment:
+1 looks like the right thing to do according to the docs:
http://docs.oracle.com/javase/7/docs/api/java/sql/Driver.html#getParentLogger()